tests/integration/components/review-actions-list/review-action/component-collection-test.ts

```diff
@@ -0,0 +1,114 @@
+        this.set('reviewAction', this.collectionSubmissionAction);
+    });
+
+    // Collection Moderator actions
+    test('Collection submission: Moderator accept action', async function(this: ThisTestContext, assert) {
+        this.collectionSubmissionAction.actionTrigger = CollectionSubmissionActionTrigger.Accept;
+
+        await render(hbs`<ReviewActionsList::ReviewAction @reviewAction={{this.reviewAction}}/>`);
+        assert.dom('[data-test-review-action-wrapper]').exists('Review action wrapper shown');
+        assert.dom('[data-test-review-action-wrapper]').containsText(
+            'Request accepted 2 days ago by moderator Superb Mario',
+            'Collection submission action wrapper shows accept string',
+        );
+        assert.dom('[data-test-review-action-comment]').doesNotExist('Empty strings do not create comment');
+    });
+
+    test('Collection submission: Moderator reject action', async function(this: ThisTestContext, assert) {
+        this.collectionSubmissionAction.actionTrigger = CollectionSubmissionActionTrigger.Reject;
+
+        await render(hbs`<ReviewActionsList::ReviewAction @reviewAction={{this.reviewAction}}/>`);
+        assert.dom('[data-test-review-action-wrapper]').containsText(
+            'Request rejected 2 days ago by moderator Superb Mario',
+            'Collection submission action wrapper shows reject string',
+        );
+    });
+
+    test('Collection submission: Moderator remove action', async function(this: ThisTestContext, assert) {
+        this.collectionSubmissionAction.actionTrigger = CollectionSubmissionActionTrigger.Remove;
+
+        await render(hbs`<ReviewActionsList::ReviewAction @reviewAction={{this.reviewAction}}/>`);
+        assert.dom('[data-test-review-action-wrapper]').containsText(
+            'Item removed 2 days ago by Superb Mario',
+            'Collection submission action wrapper shows moderator remove string',
+        );
+    });
+});
+
+module('Integration | Component | Collection | Admin | review-actions', hooks => {
+    setupRenderingTest(hooks);
+    setupIntl(hooks);
+    setupMirage(hooks);
+
+    hooks.beforeEach(async function(this: ThisTestContext) {
+        this.store = this.owner.lookup('service:store');
+        const collection = server.create('collection');
+        const mirageAction = server.create('collection-submission-action', {
+            actionTrigger: CollectionSubmissionActionTrigger.Submit,
+            creator: server.create('user', { fullName: 'Superb Mario' }, 'loggedIn'),
+            target: server.create('collection-submission', {
+                guid: server.create('node', 'withContributors'),
+                collection,
+            }),
+        });
+        this.collectionSubmissionAction = await this.store.findRecord('collection-submission-action', mirageAction.id);
+        this.set('reviewAction', this.collectionSubmissionAction);
```

Comment:
Same here. Can remove this line and change `this.reviewAction` -> `this.collectionSubmissionAction`